tools/docker/Dockerfile

```diff
@@ -57,7 +57,7 @@ RUN poetry install -E all --no-root --no-cache \
     && ${PYTHON_BIN} -m pip install gunicorn
 COPY . $SOURCES_DIR/
 RUN poetry install -E all --only-root
-RUN pip install ansible-core
+RUN pip install ansible-core==2.16
```

Comment:
Can we define a build arg with this version as default, so we can ship other versions dynamically? There will be a point when stable branches would have as default different versions

Reply:
updated